### DISCUSSION TITLE
How to make a protected area for specific users?

### DISCUSSION THREAD

**Discussion by BsNoSi** [datenstrom/community#241] [Ask a question] [answered]
What is the best way to achieve something like this with Yellow:

```
landing page (open)
subpage (open)
subpage2 (protected-a)
subpage3(protected-b)
```

where "protected" means, that there is an embedded log in "on page" for autorized users.

I think, a simple .htaccess is sufficient in most cases, I did not think about creating logins for areas or changing passwords, but this will be neccessary, I know.

**Comment by schulle4u** [comment]
Have a look at datenstrom/community#132, the 3rd comment looks interesting.

**Comment by BsNoSi** [comment]
Thanks for this hint. Is not directly, what I am thinking of, but it points in the right direction.

I must improve my PHP skills really urgent...

**Comment by markseuffert** [comment, marked as answer]
I currently use the [draft plugin](https://github.com/annaesvensson/yellow-draft) as a quick & dirty solution. For example pages that should be available only to the internal team will be marked `Status: draft`. Team members have to log in to read them. Web master can make these pages read-only by restricting user accounts, see security configuration. 

Just an idea. 😄
